bin/collator/src/local/chain_spec.rs

```diff
@@ -84,7 +84,7 @@ fn testnet_genesis(
             balances: endowed_accounts
                 .iter()
                 .cloned()
-                .map(|k| (k, 1_000_000_000_000_000_000_000))
+                .map(|k| (k, 1_000_000_000_000_000_000_000_000_000_000_000_000))
```

Comment:
why this big number? it is more than our supply

Reply:
there wasn't a particular reason, it was just to give the accounts plenty for scenarios like massive batch transfers.
If you think it's too much, I can reduce it.